Gemfile.lock

```diff
@@ -501,7 +501,8 @@ GEM
     rack (2.0.7)
     rack-attack (5.0.1)
       rack
-    rack-cors (0.4.1)
+    rack-cors (1.0.6)
```

Comment:
Wondering if update for rack and nokogiri should be in a separate PR?

Reply:
There is a separate PR for the gemfile updates but I also included them in here because they were causing a CI/CD failure for this PR https://github.com/department-of-veterans-affairs/vets-api/pull/3560/files